google/ggrc-core
# BC: GGRC-7716 Disable possibility to create mandatory GCA for Assessments on Admin page

# Comment

**Should be merged to "asmt-bulk".**

# Issue description

Disable possibility to create mandatory GCA for Assessments on Admin page

# Steps to test the changes

1. Open Custom Attributes tab on Admin page.
2. Choose Assesments object type: Mandatory column should not be shown
3. Try do add new GCA: Mandatory checkbox should not be shown
4. Try do edit GCA:  Mandatory checkbox should not be shown

# Solution description

Remove Mandatory checkbox and column for Assessments on Admin page

# Sanity checklist




# PR Review checklist
